src/components/GlobalPreferences/Notifications/NotificationModals.js

```diff
@@ -115,20 +125,18 @@ const ModalControls = styled.div`
   display: grid;
   grid-gap: ${1.5 * GU}px;
   grid-template-columns: 1fr 1fr;
-  ${breakpoint(
-    'medium',
+  ${({ large }) =>
+    large &&
     `
       display: flex;
       justify-content: flex-end;
-    `
-  )}
+    `}
 `
 
 const RemoveButton = styled(Button)`
-  ${breakpoint(
-    'medium',
+  ${({ large }) =>
+    large &&
     `
       margin-left: ${1.5 * GU}px;
-    `
-  )}
+    `}
```

Comment:
Should we make `ModalControls` and `RemoveButton` normal components, so that we can directly use `useViewport()` inside?

Reply:
Yes we can do this!